internal/mode/static/telemetry/collector.go

```diff
@@ -190,6 +200,27 @@ func collectGraphResourceCount(
 
 	ngfResourceCounts.BackendTLSPolicyCount = int64(len(g.BackendTLSPolicies))
 
+	for policyKey, policy := range g.NGFPolicies {
+		switch gvk := policyKey.GVK; gvk.Kind {
+		case kinds.ClientSettingsPolicy:
+			if policy.TargetRefs == nil {
```

Comment:
better to check for len(0) here, so we don't depend on TargetRefs being nil or not, only zero length